### DISCUSSION TITLE
E3 Gnn model used in npj paper

### DISCUSSION THREAD

**Discussion by WJiangH** [killiansheriff/ChemicalMotifIdentifier#4] [Q&A]
Dear Authors,

I have read your recent publication in *npj* with great interest and have a question regarding the use of E3-GNN for embedding chemical motifs. Specifically, is the embedding result system-insensitive? In other words, can a model trained on an FCC 5-component HEA be applied to another FCC 5-component HEA, given that the motif represents only the types of elements rather than the exact elements themselves?

Thank you for your time and assistance.

JJ

**Comment by killiansheriff** [comment]
Hi JJ, 

Yes, the embedding is totally system-insensitive: it treats atom types as numbers. If you look into the dump files provided in the [tutorial](https://github.com/killiansheriff/ChemicalMotifIdentifier/blob/main/examples/doi-10.48550-arXiv.2311.01545/fcc_ternary_system.ipynb), you will see that the information is not "element" based, it is just numbers (i.e., Cr Co Ni is type 1 2 3; but their could be any elements of your choosing!). 

Hope this helps! 

Best, 
Killian
